lib/pages/chat_details/chat_details.dart

```diff
@@ -176,7 +176,7 @@ class ChatDetailsController extends State<ChatDetails>
       future: () => room!.requestParticipants(),
     );
     if (participants.error == null) {
-      setState(() => members = participants.result);
+      setState(() => membersNotifier.value = participants.result);
```

Comment:
setState shouldn't be needed anymore if you use valueNotifier

Reply:
done